test/FunctionalTests/Framework/Workspaces/NonClrWorkspaces/NonClrService/NonClrExpressionTreeToXmlSerializer.cs

```diff
@@ -69,7 +69,7 @@ internal class ExpressionTreeToXmlSerializer : System.Data.Test.Astoria.LateBoun
         public static String Serialize(Expression exp)
         {
             outputText = new StringBuilder();
-            writer = XmlTextWriter.Create(outputText);
+            writer = XmlWriter.Create(outputText);

```

Comment:
>XmlWriter.Create(outputText); [](start = 21, length = 29)

doesn't seem like this change is related to nav prop bindings. Code expression simplification instead?

Reply:
This was a bug that prevent compilation on certain platforms.

---
In reply to: [175303989](https://github.com/OData/odata.net/pull/1109#discussion_r175303989) [](ancestors = 175303989)